authors/apps/authentication/serializers.py

```diff
@@ -15,25 +15,26 @@ class RegistrationSerializer(serializers.ModelSerializer):
         min_length=8,
         write_only=True
     )
-
     # The client should not be able to send a token along with a registration
     # request. Making `token` read-only handles that for us.
 
     class Meta:
         model = User
         # List all of the fields that could possibly be included in a request
         # or response, including fields specified explicitly above.
-        fields = ['email', 'username', 'password']
+        fields = ['email', 'username', 'password', 'message']
```

Comment:
@Emmanuelaaron is the message attribute a requirement for the user model?

Reply:
The significance is to get a customized message advising the user to check their email to verify their account after signup

Reply:
It's significance is to output a customized message when a user signs up as recommended by the TTL